stable/insights-agent/README.md

```diff
@@ -49,7 +49,7 @@ cluster's access token.
 
 To better serve customers with a large number of clusters, we've created a flow
 that allows you to easily deploy the Insights Agent across your fleet.
-[Read more here](https://insights.docs.fairwinds.com/run/agent/installation/#fleet-installation)
+[Read more here](https://insights.docs.fairwinds.com/features/in-cluster-scanning/#fleet-installation)
```

Comment:
Alternately, this could be fixed on the docs, providing a redirect [here](https://github.com/FairwindsOps/insights-docs/blob/main/docs/.vuepress/enhanceApp.js)